metadata/latest/Azure.AI.Vision.ImageAnalysis.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Name": "Azure.AI.Vision.ImageAnalysis ",
+    "Version": "1.11.1-beta.1",
```

Comment:
This file should go in the metadata/preview/ folder because it has a preview version.